core/tests/protractor.conf.js

```diff
@@ -376,10 +376,10 @@ exports.config = {
         ' because it\'s on CircleCI');
     }
 
-    // Screenshots will only run on CircleCI, since we don't have videos here.
-    // We don't need these on Github Actions since we have videos.
+    // Screenshots will only run on the develop branch of "oppia/oppia", since
+    // currently we don't run videos due to memory issues.
 
-    if (process.env.CIRCLECI) {
+    if (process.env.REPOSITORY == 'oppia/oppia' && process.env.GITHUB_REF == 'refs/heads/develop') {
```

Comment:
Since there aren't memory issues with screenshots, can we run them all the time?

Reply:
^ may be worth to do this, since it will increase ease of debugging for developers